src/assets/mixins.styl

```diff
@@ -0,0 +1,29 @@
+//Fonts mixins
+setFont($family, $weight)
+  if $family == 'playfair'
+    font-family 'Playfair Display', serif
+  if $family == 'source'
+    font-family 'Source Sans Pro', sans-serif
+  font-weight $weight
+
+//Grid mixin
+makeGrid($cols, x=25px, y=25px)
```

Comment:
rename all of the mixins to use dash instead of camelcase [Stylus mixins](http://stylus-lang.com/docs/mixins.html) is a more common practice to do it this way